templates/winserver2019/dist/dev.dockerfile.j2

```diff
@@ -17,7 +18,7 @@ RUN powershell.exe -Command `
 RUN powershell.exe -Command `
     Invoke-WebRequest -URI https://files.pythonhosted.org/packages/7f/e6/1639d2de28c27632e3136015ecfd67774cca6f55146507baeaef06b113ba/pypi-kenlm-0.1.20190403.tar.gz -OutFile %INTEL_OPENVINO_DIR%\\deployment_tools\\model_optimizer\\pypi-kenlm.tar.gz
 {% endif %}
-
+{% endif %}
```

Comment:
Add chardet and flake8-import-order downloading for dev, data_dev, proprietary windows images